### PR TITLE
Return empty array if user_meta is empty or false

### PR DESCRIPTION
**Pull request by lmansur**
https://3.basecamp.com/3293071/buckets/5610905/todos/4513433623

### CODE REVIEW

**Comment by lmansur** [comment]
@mrhead @northerner @jcbpl mind taking a look at this one? I'd like to release this fix today if possible

File: wordpress/wp-content/plugins/memberful-wp/src/acl.php [hunk 1]
```diff
@@ -129,7 +129,21 @@ function memberful_wp_user_feeds($user_id) {
...
+
+/**
+ * `get_user_meta` may return `false` or `""` if the given $user_id
+ * doesn't exist or is invalid. Because the expected ACL interface is an array,
+ * we return an empty one.
+ */
+function memberful_wp_get_user_meta_for_acl($user_id, $meta_key, $single = TRUE) {
+  $meta = get_user_meta($user_id, $meta_key, $single);
+
+  if ($meta == false || $meta == "")
```
Comment (mrhead) [inline wordpress/wp-content/plugins/memberful-wp/src/acl.php:143]: According to [PHP's type comparison tables](https://www.php.net/manual/en/types.comparisons.php), `"" == false` is `true` so we can use:

```suggestion
  if ($meta == false)
```

**Review by mrhead — APPROVED** [review]
I'm sorry, @lmansur, I've missed a notification about this PR.

It looks good to me. I had one non-blocking comment.

File: wordpress/wp-content/plugins/memberful-wp/src/acl.php (hunk 1 above)
Comment (lmansur) [reply wordpress/wp-content/plugins/memberful-wp/src/acl.php:143]: Good to know, thanks!

**Pull request merged**